hf_demo_2x.py

```diff
@@ -0,0 +1,54 @@
+import cv2
+import math
+import sys
+import torch
+import numpy as np
+import argparse
+from imageio import mimsave
+
+'''==========import from our code=========='''
+sys.path.append('.')
+import config as cfg
+from Trainer_finetune import Model
+from benchmark.utils.padder import InputPadder
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--model', default='VFIMamba_S', type=str)
+parser.add_argument('--scale', default=0, type=float)
+
+args = parser.parse_args()
+assert args.model in ['VFIMamba_S', 'VFIMamba'], 'Model not exists!'
+
+
+'''==========Model setting=========='''
+TTA = False
+if args.model == 'VFIMamba':
+    TTA = True
+    cfg.MODEL_CONFIG['LOGNAME'] = 'VFIMamba'
+    cfg.MODEL_CONFIG['MODEL_ARCH'] = cfg.init_model_config(
+        F = 32,
+        depth = [2, 2, 2, 3, 3]
+    )
```

Comment:
Could these config values be stored in a `config.json` file alongside the model weights on the Hub? This way they could be downloaded when instantiating the model and therefore reduce the boilerplate code to use `Model`. 

(just a suggestion, I'm not entirely sure how `cfg` and `Model` are interacting currently)

Reply:
Absolutely! Thanks to your advice, we can now directly initialize a model when loading weights from Hugging Face using the following code (as also demonstrated in hf_demo_2x.py): 
```python
model = Model.from_pretrained(args.model)
```
Cheers!

Reply:
Awesome!